docs/CONTRIBUTING.md

```diff
@@ -91,10 +91,11 @@ developer, it will be merged into the source tree.
 The following rules shall be applied strictly for the `amethyst` repository. For other repositories of this organization,
 thorough review would be desirable, but no strict application is required there due to lower activity and less influence.
 
-* Pull Requests shall not be merged before at least 24 hours have passed.
 * Pull Requests shall be approved by at least two members (two approvals from contributors can count as one member approval.)
-* Merging a PR shall be done with `bors r+`
-* If a member self-requested a review, the PR shall not be merged until they reviewed the PR (exception: the member becomes inactive)
+* Pull Requests opened by organization members may be merged if approved by one member.
+* Merging a PR shall be done with `bors r+`, if possible.
```

Comment:
Can we add more here? I would like a link to the docs and if there are more than on approver it should be `bors r=<first>, <second>,...`. We can then link to bors commands doc.